hack/windows/Install-OVS.ps1

```diff
@@ -17,13 +17,18 @@
   .PARAMETER LocalFile
   Specifies the path of a local OpenvSwitch package to be used for installation.
   When the param is used, "DownloadURL" and "DownloadDir" params will be ignored.
+
+  .PARAMETER ImportCertificate
+  Specifies if a certificate file is needed for OVS package. If true, certificate
+  will be got from OVSExt.sys and a package.cer file will be generated.
```

Comment:
s/will be got/will be retrieved

Reply:
Updated. Installation instruction is also updated about the newly introduced option.